boards/xtensa/m5stack_core2/m5stack_core2_defconfig

```diff
@@ -4,7 +4,7 @@ CONFIG_BOARD_M5STACK_CORE2=y
 
 CONFIG_SOC_SERIES_ESP32=y
 
-CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_ESP_HEAP_MEM_POOL_REGION_1_SIZE=0
```

Comment:
If the change of the main stack size is needed by the lvgl sample maybe adding a .conf in the sample instead would make sense? This increases the footprint also for non lvgl applications.

Reply:
Thank you. You are absolutely right!
Added a config file on app level.